docs/setuptools.txt

```diff
@@ -2142,6 +2148,12 @@ distutils configuration file the option will be added to (or removed from).
 ``test`` - Build package and run a unittest suite
 =================================================
 
+.. warning::
+    **test** is deprecated in favor of using `tox
+    <https://tox.readthedocs.org/>`_, `unittest
```

Comment:
Should be https://tox.readthedocs.io

Also, I think we can reword this a bit - I'd prefer it to be clear that `tox` is the actual solution here, and the other two are test runners (plus `pytest` should probably be prioritized over `unittest`), I'd word it as:

```rst
.. warning::
    ``test`` is deprecated in favor of using `tox
    <https://tox.readthedocs.io>`_, or directly invoking your test runner (e.g.
    `pytest <https://docs.pytest.org>`_ or `unittest
    <https://docs.python.org/library/unittest.html>`_).
```

Another wording that might put our thumbs on the scale in favor of `tox` a bit more lightly would be:

```rst
.. warning::
  ``test`` is deprecated and will be removed in a future version. Users looking
  for a generic test entry point independent of test runner are encouraged to
  use `tox <https://tox.readthedocs.io>`_.
```